src/systemtraynotificationhandler.cpp

```diff
@@ -245,8 +255,7 @@ void SystemTrayNotificationHandler::showHideWindow() {
   }
 }
 
-void SystemTrayNotificationHandler::maybeActivated(
-    QSystemTrayIcon::ActivationReason reason) {
+void SystemTrayNotificationHandler::maybeActivated(int reason) {
```

Comment:
why `int` and not `QSystemTrayIcon::ActivationReason` ?

Reply:
I see! It's for the include. I think we should keep the include<QSystemTrayIcon> and make the code more readable.

Reply:
If we keep the include we can't import this module in tests :/